app/controllers/authentication.js

```diff
@@ -20,9 +20,22 @@ class AuthenticationController {
   constructor() {
     this.config = {
       GOOGLE_SECRET: nconf.get('google_secret'),
-      TOKEN_SECRET: nconf.get('webtoken'),
+      TOKEN_SECRET: nconf.get('webtoken')
     };
   }
+  
+  static generateEmail(name) {
+    // @todo more robust mechanism needed
+    // This assumes that name is "<first-name> (<second-name> )<last-name>"
+    // and EMAIL_DOMAIN is specified
+    let parts = _.map(name.split(" "), _.toLower);
+    if(parts.length<2) {
+      return '';
+    }
+    let email = `${parts[0]}.${parts[parts.length-1]}@${emailDomain}`;
```

Comment:
Spaced infix operators, e.g. "parts.length - 1", we really should start enforcing a specified set of eslint rules. Personally I like the spaced infix operators as they make it easier to spot operators and subsequently split the line into logical blocks.